### PR TITLE
Satellite 6.4, 6.5 is supposed to use rhel-7-server-ansible-2.6-rpms repo

### PR DESCRIPTION
**Pull request by jhutar**
See https://access.redhat.com/documentation/en-us/red_hat_satellite/6.4/html-single/installing_satellite_server_from_a_connected_network/index#configuring_repositories_satellite

### CODE REVIEW

File: automation_tools/repository.py [hunk 1]
```diff
@@ -145,14 +146,14 @@ def enable_satellite_repos(cdn=False, beta=False, disable_enabled=True,
...
-        disable_repos('*', silent=True)
+        disable_repos('*', silent=True, warn_only=True)
 
     repos = [
         'rhel-{0}-server-rpms',
         'rhel-server-rhscl-{0}-rpms',
     ]
     if sat_version == '6.4':
-        repos.append('rhel-{0}-server-ansible-2-rpms')
+        repos.append('rhel-{0}-server-ansible-2.6-rpms')
```
Comment (ntkathole) [inline automation_tools/repository.py]: @jhutar please add 6.5, we need ansible 2.6 for 6.5 as well.

**Comment by ntkathole** [comment]
I see same commit in another PR as well. PR #766 . Do you want to close other?

**Comment by jhutar** [comment]
Hello @ntkathole . No, these are two different commits. I was just lazy to fix this in case first one would be merged anyway

**Comment by jhutar** [comment]
The other commit removed

File: automation_tools/repository.py (hunk 1 above)
Comment (jhutar) [reply automation_tools/repository.py]: Added if for 6.5. Thanks

**Comment by lpramuk** [comment]
Obsoleted by PR #772
or at least refactor your PR so that match changes there

**Comment by jhutar** [comment]
After discussion with lpramuk accommodated to changes from PR #772 and added if for Satellite 6.6 (that one needs Ansible 2.8, but it was not released yet, so putting there generic Ansible 2 repo).

**Comment by jhutar** [comment]
Just for a record: just ran on 6.6@RHEL7 and it did not exploded:

```
[satellite.lan.local] run: subscription-manager repos --disable "*"
rhel 7 6
[satellite.lan.local] run: subscription-manager repos --enable "rhel-7-server-rpms" --enable "rhel-server-rhscl-7-rpms" --enable "rhel-7-server-ansible-2-rpms" --enable "rhel-7-server-satellite-maintenance-6-rpms"
[satellite.lan.local] out: Repository 'rhel-7-server-rpms' is enabled for this system.                                     
[satellite.lan.local] out: Repository 'rhel-7-server-satellite-maintenance-6-rpms' is enabled for this system.
[satellite.lan.local] out: Repository 'rhel-7-server-ansible-2-rpms' is enabled for this system.
[satellite.lan.local] out: Repository 'rhel-server-rhscl-7-rpms' is enabled for this system.
```

**Pull request merged**